src/tree/tree.scss

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$iui-active-outline: thin solid t(iui-color-foreground-primary);
+$iui-expander-inline-padding: $iui-component-offset + $iui-xxs + 1px; // margin + padding + border
+$iui-expander-button-width: ($iui-icons-default) + ($iui-expander-inline-padding * 2); // icon width + inline padding on both sides
+
+@mixin iui-tree {
+  @include iui-reset;
+  list-style: none;
+}
+
+@mixin iui-sub-tree {
+  @include iui-reset;
+  list-style: none;
+}
+
+@mixin iui-tree-node {
+  display: flex;
+  cursor: pointer;
+  padding: 0 $iui-s;
+
+  &-checkbox {
+    margin-right: $iui-s;
+  }
+
+  &-content {
+    display: flex;
+    align-items: center;
+    box-sizing: border-box;
+    min-height: $iui-baseline * 3;
+    margin-left: calc(#{$iui-expander-button-width} * (var(--level, 0)));
+    overflow: hidden;
+    padding-left: $iui-xxs;
+
+    &-icon {
+      @include iui-icons-default;
+      padding: 0 $iui-expander-inline-padding;
+      flex-shrink: 0;
+
+      &:first-child {
+        margin-left: $iui-expander-button-width;
+      }
+    }
+
+    &-expander-icon {
+      @media (prefers-reduced-motion: no-preference) {
+        transition: transform $iui-speed-fast ease-out;
+      }
+
+      &-expanded {
+        transform: rotate(90deg);
+      }
+    }
+
+    &-label {
+      min-width: 0;
+      padding-left: $iui-expander-inline-padding;
+
+      &:first-child {
+        margin-left: $iui-expander-button-width;
+      }
+    }
+
+    &-title,
+    &-caption {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+
+    &-title {
+      font-size: $iui-font-size;
+    }
+
+    &-caption {
+      font-size: $iui-font-size-small;
+      @include themed {
+        color: t(iui-text-color-muted);
+      }
+    }
+  }
+
+  &:focus {
+    @include themed {
+      outline: thin solid rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-4));
+      outline-offset: -1px;
+    }
+  }
+
+  &:focus:not(:focus-visible) {
+    outline-offset: -2px;
+  }
+
+  &:hover {
+    @include themed {
+      background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-6));
+    }
+
+    .iui-tree-node-content-caption {
+      @include themed {
+        color: t(iui-color-foreground-body);
+      }
+      transition: color $iui-speed-fast ease;
+    }
+  }
+
+  &.iui-active {
+    @include themed {
+      background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
+      outline: $iui-active-outline;
+      outline-offset: -1px;
+    }
+
+    &:focus {
+      outline-width: $iui-xxs;
+      outline-offset: -2px;
+    }
+  }
+
+  &.iui-disabled {
+    cursor: not-allowed;
+    outline: none;
+    background-color: transparent;
```

Comment:
I removed `background-color` from the tree SCSS and placed it in the `<head><style>`.  To avoid `.iui-disabled` from getting a hover state, I changed the `background-color` to `transparent`.  An alternative would be `&:hover:not(.iui-disabled)` but I imagine that has specificity issues.